concourse/tasks/delete_admin.yml

```diff
@@ -31,3 +32,6 @@ run:
       echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
 
       cf delete-user "${NAME}" -f
+
+      echo "Removing leaked users"
```

Comment:
I'd say st. like "Purging any ${PREFIX} leftover users". Just to put the prefix into the output so that it is clear that it's only removing those users.